src/main/java/org/folio/inventory/validation/experimental/UnavaliableTargetItemStatusValidator.java

```diff
@@ -0,0 +1,20 @@
+package org.folio.inventory.validation.experimental;
+
+import org.folio.inventory.domain.items.ItemStatusName;
+
+import static java.util.Set.of;
+
+public class UnavaliableTargetItemStatusValidator extends AbstractTargetItemStatusValidator {
```

Comment:
I think there is a typo in `unavailable`

```suggestion
public class UnavailableTargetItemStatusValidator extends AbstractTargetItemStatusValidator {
```